packages/framework/src/Model/Seo/HreflangLinksFacade.php

```diff
@@ -207,7 +239,7 @@ protected function doGetHrefLinks(
      * @param int $currentDomainId
      * @return int[]
      */
-    protected function getRelevantDomainIds(int $currentDomainId): array
+    public function getRelevantDomainIds(int $currentDomainId): array
```

Comment:
this does not need to be changed to public, it is still used within this class only

Reply:
Yes, that changed after some refactoring :)